python/translit/converters/translit_cyr_taras_to_lat.py

```diff
@@ -1,4 +1,6 @@
 import re
+from translit import diff_mode
```

Comment:
I do not see place where you use `diff_mode` in this file.

Reply:
just worry about mixing library and testing code

Reply:
Oops, it was set to auto-merge. I'll fix it in the other PR.